### DISCUSSION TITLE
Output file name

### DISCUSSION THREAD

**Discussion by xstrex** [supermerill/SuperSlicer#3433] [Q&A] [answered]
Just wondering if there's a way to change the default file name output? 
Appending `_32m_0.28mm_205C_PLA_ENDER3V2.gcode` is just complicating things, and if possible I'd like to just turn it off, so `foobar.stl` outputs to `foobar.gcode`.

**Comment by neophyl** [comment, marked as answer]
Yes of course.  That's been possible since forever.  
What is output is defined in the Print Profile.  So Print Settings>Output Options>Output File>Output Filename format:

For example {input_filename_base}_{layer_height}mm_{filament_type[0]}_{printer_model}.gcode gives you what you listed .  Just remove everything except the input filename base and then save the profile.  Then whenever you slice using that profile you will only get the output name you want.

You do have to update all your profiles though as it is not a global thing.

**Reply by xstrex** [reply]
Great, thanks so much, I thought it would be an easy fix, just couldn't find documentation or reference to it.

**Comment by neophyl** [comment]
Btw some documentation is available here https://help.prusa3d.com/article/list-of-placeholders_205643
